src/components/[guild]/hooks/useIsMember.ts

```diff
@@ -1,13 +1,11 @@
-import useMemberships from "components/explorer/hooks/useMemberships"
-import useGuild from "components/[guild]/hooks/useGuild"
+import useMembership from "components/explorer/hooks/useMemberships"
 
 const useIsMember = (): boolean => {
```

Comment:
This appears to be doing the same as hasAccess in useMembership now, so we could remove this hook and just use that (IMO renamed to isMember, since that's more accurate than just having access)

Reply:
Good idea, changed in https://github.com/guildxyz/guild.xyz/pull/1048/commits/353d7d0d7e82e12bac9af3e1511b22a530b19d95!